conda-store-server/conda_store_server/_internal/schema.py

```diff
@@ -446,80 +442,6 @@ def __str__(self):
         return str(self.dict())
 
 
-###############################
-#  Docker Registry Schema
-###############################
-
-
-def _docker_datetime_factory():
-    """Utcnow datetime + timezone as string"""
-    return datetime.datetime.utcnow().astimezone().isoformat()
-
-
-class DockerManifestLayer(BaseModel):
-    mediaType: str = "application/vnd.docker.image.rootfs.diff.tar.gzip"
-    size: int
-    digest: str
-
-
-class DockerManifestConfig(BaseModel):
-    mediaType: str = "application/vnd.docker.container.image.v1+json"
-    size: int
-    digest: str
-
-
-class DockerManifest(BaseModel):
-    schemaVersion: int = 2
-    mediaType: str = "application/vnd.docker.distribution.manifest.v2+json"
-    config: DockerManifestConfig
-    layers: List[DockerManifestLayer] = []
-
-
-class DockerConfigConfig(BaseModel):
-    Hostname: str = ""
-    Domainname: str = ""
-    User: str = ""
-    AttachStdin: bool = False
-    AttachStdout: bool = False
-    AttachStderr: bool = False
-    Tty: bool = False
-    OpenStdin: bool = False
-    StdinOnce: bool = False
-    Env: List[str] = [
-        "PATH=/opt/conda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
-    ]
-    Cmd: List[str] = ["/bin/sh"]
-    ArgsEscaped: bool = True
-    Image: Optional[str] = None
-    Volumes: Optional[List[str]] = None
-    WorkingDir: str = ""
-    Entrypoint: Optional[str] = None
-    OnBuild: Optional[str] = None
-    Labels: Optional[Dict[str, str]] = {"conda_store": "0.0.1"}
-
-
-class DockerConfigRootFS(BaseModel):
-    type: str = "layers"
-    diff_ids: List[str] = []
-
-
-class DockerConfigHistory(BaseModel):
-    created: str = Field(default_factory=_docker_datetime_factory)
-    created_by: str = ""
-
-
-class DockerConfig(BaseModel):
-    architecture: str = "amd64"
-    os: str = "linux"
-    config: DockerConfigConfig
-    container: str
-    container_config: DockerConfigConfig
-    created: str = Field(default_factory=_docker_datetime_factory)
-    docker_version: str = "18.09.7"
-    history: List[DockerConfigHistory] = []
-    rootfs: DockerConfigRootFS
-
-
 # https://docs.docker.com/registry/spec/api/#errors-2
 class DockerRegistryError(enum.Enum):
```

Comment:
Is it in scope to remove this as well?

Reply:
I think we should keep this for now. This is used in the docker registry whenever there are errors. We are deprecating this functionality in pr #1016. But, we'll need to keep it around until the registry is fully removed.